### DISCUSSION TITLE
How to keep up to date with template updates?

### DISCUSSION THREAD

**Discussion by SvenMeyer** [PaulRBerg/hardhat-template#58] [Q&A]
@paulrberg I really like to keep my projects up to date with updates of the template.
Is there an "easy way" to do that ?
Thanks a lot for the good work !

**Comment by PaulRBerg** [comment]
I can't think of anything except the good old rebase method, but that is likely to lead you to git conflicts. I would instead recommend looking at the [git history](https://github.com/paulrberg/solidity-template/commits/main) and cherry-picking the changes that you like most, then manually implementing them on your fork.

At any rate, because this is a template, every time you start a new project, you should get the latest updates.

**Reply by SvenMeyer** [reply]
Thanks. That's how I am actually doing it atm, "looking at the git history and cherry-picking the changes that you like most, then manually implementing them on your fork" and always grabbing a fresh template for every new project ... but I thought that maybe somebody (you) has a smarter approach ;-)
Sometimes I even create a new project from the template, copy the contracts and test folders from my existing project, and then I am almost done ...
